users/templates/users/user_login.html

```diff
@@ -36,6 +36,9 @@
                 <button class="btn btn-lg btn-raised btn-material-light-blue" type="submit">Login</button>
             </form>
 
+            <br/>
+            <p><a href="http://localhost:8000{% url 'users:request_password_reset' %}">forgot password?</a></p>
```

Comment:
Also don't hardcode `http://localhost:8000`. Just use the `{% url ... %}`. Once I install it on a webserver with a different address, this will fail.